test/fixers/test_hackfmt_fixer_callback.vader

```diff
@@ -0,0 +1,38 @@
+Before:
+  Save g:ale_php_hackfmt_executable
+  Save g:ale_php_hackfmt_options
+
+  " Use an invalid global executable, so we don't match it.
+  let g:ale_php_hackfmt_executable = 'xxxinvalid'
+  let g:ale_php_hackfmt_options = ''
+
+  call ale#test#SetDirectory('/testplugin/test/fixers')
+
+After:
+  Restore
+
+  call ale#test#RestoreDirectory()
+
+Execute(The hackfmt callback should return the correct default values):
+  call ale#test#SetFilename('../hack_files/testfile.php')
+
+  AssertEqual
+  \ {
+  \   'read_temporary_file': 1,
+  \   'command': ale#Escape('xxxinvalid')
+  \     . ' %t',
+  \ },
+  \ ale#fixers#hackfmt#Fix(bufnr(''))
+
+Execute(The hackfmt callback should include custom hackfmt options):
+  let g:ale_php_hackfmt_options = "--skip-children"
+  call ale#test#SetFilename('../hack_files/testfile.php')
+
+  AssertEqual
+  \ {
+  \   'read_temporary_file': 1,
+  \   'command': ale#Escape('xxxinvalid')
+  \     . ' ' . g:ale_php_hackfmt_options
```

Comment:
Write a string literal here instead, rather than repeating the variable.

Reply:
Will do!